app/views/collection_templates/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<div class="row row-content">
+  <section>
+    <h2><%= current_user.provider_id %> <%= plural_resource_name.titleize %></h2>
+
+    <%= link_to "Create a #{resource_name.titleize}", send("new_#{resource_name}_path"), class: 'eui-btn--blue space-bot' %>
+
```

Comment:
Everything beyond line 5 in this file is likely to change, in whole or in part, in subsequent tickets.  

Minimally, the plan is to make action column have two sub columns that contain "Edit" and "Delete" with appropriate links.  Additionally, the titles will link to the show pages, not to making a draft.